.pre-commit-config.yaml

```diff
@@ -12,24 +12,22 @@ exclude: >
         ^src/sentry/data/
     )
 repos:
--   repo: https://github.com/ambv/black
-    rev: 19.10b0
+-   repo: local
     hooks:
+    # Configuration for black exists in pyproject.toml.
     - id: black
+      name: black
+      entry: black
+      language: python
       types: [python]
-      exclude: (migrations/)
--   repo: local
```

Comment:
I'm trying to follow along. Did you mainly upgrade packages (and adjusted flags) and move the `local` repo at the top?

Reply:
The configuration is consolidated in `pyproject.toml` for black and `setup.cfg` for flake8. And yes to everything.